examples/password_strength/README.md

```diff
@@ -0,0 +1,33 @@
+# Boids Example
```

Comment:
This file is to-do.

Reply:
I kind of forgot about this until now because of this outstanding issue. Is there anything blocking this or are you still working on it?

Reply:
Should be ready now.